reciprocalspaceship/dataset.py

```diff
@@ -1287,31 +1287,83 @@ def canonicalize_phases(self, inplace=False):
 
         return self
 
-    def to_reciprocalgrid(self, key, gridsize):
+    def get_gridsize(self, dmin=None, sample_rate=3.0):
```

Comment:
I propose to change the name of this method to `get_fourier_gridsize` or `get_reciprocal_gridsize` something similar. Otherwise, it is not apparent what the "grid" is. I think the is the same reason it's called `ds.to_reciprocalgrid` and not simply `ds.to_grid`.

Reply:
This is a nitpick but i think `gridsize` should be `grid_size` in parameter and method names. I don't see why we would contract it.

Reply:
updated call signature and method name